code/script.js

```diff
@@ -1,19 +1,100 @@
-// Start here
+//This script is using else.if. The script switch.js is connected with html where some switch and ternary operator is added.
 
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
+);
+
+const username = prompt("What's your name?", "Confused Developer");
+
+alert("Welcome! Let's order some yummy food " + username + "!");
 
 // Step 2 - Food choice
-// Your code goes here
+let choosefood = prompt("What would you like to order? 1 = Pizza, 2 = Pasta, 3 = Salad", "Type your number of choice here");
+
+let foodName;
+
+if (choosefood === "1") {
+  foodName = "Pizza";
+} else if (choosefood === "2") {
+  foodName = "Pasta";
+} else if (choosefood === "3") {
+  foodName = "Salad";
+} else {
+  alert(`Sorry, invalid number/choice. Please start over.`); process.exit(1);
+}
+
+alert("Yummy! " + foodName + "! What a great choice!");
+
+// Step 3 - Subtype choice depending on food choice
+let specifyfood;
+
+if (foodName === "Pizza") {
+  specifyfood = prompt("Select a type of pizza: 1 = Spicy pizza, 2 = Yummy Pizza, 3= Yummy yummy pizza")
```

Comment:
One tip is something I learned when coding my pizzeria, if you want your text to be on several lines you can put "\n" right before the word that you want to get it on the next line, like this:

`specifyfood = prompt("Select a type of pizza:\n1 = Spicy pizza\n2 = Yummy Pizza\n3= Yummy yummy pizza")`

Reply:
Thank you! Will try that!